### DISCUSSION TITLE
Owon PC321 Missing identities and clamps

### DISCUSSION THREAD

**Discussion by Anthael** [zigpy/zha-device-handlers#2745] [Q&A]
Hi

I decided to move from jeedom to home assistant ( OS ) and i m using a conbee II with a power clamp with 3 phases ( OWON PC321 ).
I was running Zigbee2Mqtt on jeedom and I m trying to get working with ZHA 

The last version display only 1 clamp on the 3 available.

How can i locate the config files or may be possible to modify the actual version to include missing data ?

Thanks by advance

**Comment by MattWestb** [comment]
If running Supervided HA or OS install the _Home Assistant Add-on: File editor_ and you can edit the HA `config.yaml `easy with it and other files.

**Comment by Hedda** [comment]
> The last version display only 1 clamp on the 3 available.

It a device is not showing all expected entities under ZHA then it probably needs a quirk (ZHA Device Handler) so do submit a new "Device support request" issue under Issues -> https://github.com/zigpy/zha-device-handlers/issues/

Edit ZHA docs if unclear -> https://www.home-assistant.io/integrations/zha#how-to-add-support-for-new-and-unsupported-devices
